openmc/mesh.py

```diff
@@ -215,6 +215,7 @@ def from_hdf5(cls, group):
         mesh.lower_left = group['lower_left'][()]
         mesh.upper_right = group['upper_right'][()]
         mesh.width = group['width'][()]
+        mesh.length_multiplier = group['length_multiplier'][()] if 'length_multiplier' in group else 1.0
```

Comment:
We'll want to put this in the `UnstructuredMesh.from_hdf5` method.

While I typically like these kind of inline if-else statements, we can take out the `else` since the `length_multiplier is already 1.0 due to the default arg when the new mesh class is created above.

```suggestion
    if 'length_multiplier' in group:
        mesh.length_multiplier = group['length_multiplier'][()] 
```

Reply:
Unless I messed up in pulling your most recent changes, it looks like these lines still need to be moved to the `UnstructuredMesh.from_hdf5` method

Reply:
I think I added it? I see it on the "Files Changed" tab at least. Your batch suggestion didn't have the right indentation, so I had to change it locally after adding it to the batch. Maybe that did something strange with viewing the changes?

Reply:
Oh I'm dumb. No that's my bad - I blame Python's lack of `ClassName::method_name` for why I didn't see that!